app/moves/middleware.js

```diff
@@ -74,13 +81,20 @@ module.exports = {
     }
 
     try {
-      const userLocations = get(req.session, 'user.locations', [])
-      const locationIds =
-        fromLocationId || userLocations.map(location => location.id).join(',')
-      const [requestedMoves, cancelledMoves] = await Promise.all([
-        moveService.getRequested({ moveDate, fromLocationId: locationIds }),
-        moveService.getCancelled({ moveDate, fromLocationId: locationIds }),
-      ])
+      const locationIds = fromLocationId
+        ? fromLocationId.split(',')
```

Comment:
Does this need to be split? I thought the logic around `fromLocationId` in  the session  is that it will only ever be one. So if set, only display moves from that location, if not, display all moves from the locations 
 that the user has access - which will be an array.